tests/console/validate_raid_encrypt_home.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate that a RAID multi-disk is directly formatted with an
+# encrypted home (md is not partitioned).
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use Mojo::JSON qw(decode_json);
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub run {
+    select_console 'root-console';
+    my $md = get_test_suite_data()->{mds}[1];
+
+    my $lsblk_output = decode_json(script_output("lsblk -M -J /dev/$md->{name}"));
+    my $dev_name = $lsblk_output->{blockdevices}[0]{name};
+    my $dev_blockdevices = $lsblk_output->{blockdevices}[0];
+    my $blockdevice_directly_formatted = $dev_blockdevices->{children}[0];
+    my $dev_children_name = $blockdevice_directly_formatted->{name};
+    my $dev_children_type = $blockdevice_directly_formatted->{type};
+    my $dev_children_mountpoint = $blockdevice_directly_formatted->{mountpoints}[0];
+    my $actual = $dev_name . ' ' . $dev_children_type . ' ' . $dev_children_mountpoint;
```

Comment:
you need a data structure here and not a hack with string with the element separated by spaces, use an array please.